packages/explorer/src/bin/explorer.ts

```diff
@@ -97,6 +101,30 @@ async function startExplorer() {
   }
 }
 
+async function startStoreIndexer() {
+  if (chainId !== anvil.id) {
+    console.log("Skipping SQLite indexer for non-anvil chain ID", chainId);
+    return;
+  }
```

Comment:
thinking it might be cool to accept an RPC URL rather than a chain ID, then we can pass that straight through to the indexer

if we need the chain ID, we can always ask the RPC for it

Reply:
(this prob would require deeper changes in explorer itself to also be able to talk to the provided rpc, so def out of scope of this PR)

Reply:
agreed, will do that separately once we start to focus on any chain support